brave/brave-core
# Adds processor to transactions

Resolves https://github.com/brave/brave-browser/issues/8986

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- claim grant
- connect to KYC uphold
- set AC amount to be more than claimed grant
- add verified publisher to AC table
- wait for AC to trigger
- go to monthly report and make sure that in transaction you see source in parenthesis 

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
